src/index.js

```diff
@@ -1,40 +1,84 @@
 import './style.css';
 
-const tasks = [
-  {
-    description: 'Take a Shower',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Cook today',
-    completed: false,
-    index: 10,
-  },
-  {
-    description: 'Buy groceries',
-    completed: false,
-    index: 3,
-  },
-  {
-    description: ' Homework',
-    completed: false,
-    index: 8,
-  },
-  {
-    description: ' List Project',
-    completed: false,
-    index: 4,
-  },
-];
+/*
+ localStorage.setItem('tasks', JSON.stringify(tasks));
+*/
+
+const tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+
+// Add an element
+const addTask = document.getElementById('addItem');
+addTask.addEventListener('keypress', (e) => {
+  if (e.key === 'Enter') {
+    e.preventDefault();
+    window.location.reload();
+    const maxId = tasks.length;
+    tasks.push({
+      description: addTask.value,
+      Completed: false,
+      index: maxId,
+    });
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+    window.location.reload();
+  }
+});
 
 const show = () => {
   const getList = document.getElementById('todo-list');
   for (let i = 0; i < tasks.length; i += 1) {
     const item = document.createElement('li');
-    item.innerHTML = `<input type="checkbox" class="check" />  ${tasks[i].description}`;
+    let chek = '';
+    if (tasks[i].Completed === true) {
+      chek = 'checked';
+    }
+    item.innerHTML = `<input  type="checkbox" ${chek} class="check" onchange=checkboxx(${i}) />  ${tasks[i].description}
+    <button onclick=deletebtn(${tasks[i].index}) class="deletebtn">Delete</button>`;
     getList.appendChild(item);
   }
 };
 
-show();
\ No newline at end of file
+show();
+
+// remove an element
+window.deletebtn = (index) => {
+  const result = tasks.filter((item) => item.index !== index);
+  const newtasks = result;
+  for (let i = 0; i < newtasks.length; i += 1) {
+    newtasks[i].index = i;
+  }
+  localStorage.setItem('tasks', JSON.stringify(newtasks));
+  window.location.reload();
+};
+
+// checket
+window.checkboxx = (i) => {
+  if (tasks[i].Completed === false) {
+    tasks[i].Completed = true;
+  } else { tasks[i].Completed = false; }
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+};
+
+// reindex
+const reIndex = () => {
+  for (let i = 0; i < tasks.length; i += 1) {
+    tasks[i].index = i;
```

Comment:
Please make sure your index is indexed correctly like on the project requirements. 🦾
<img width="698" alt="Screenshot 2023-01-29 at 1 23 06 AM" src="https://user-images.githubusercontent.com/25887253/215293794-e1c828f3-34d3-412b-921c-27655b1e510b.png">

<img width="374" alt="Screenshot 2023-01-29 at 1 21 43 AM" src="https://user-images.githubusercontent.com/25887253/215293742-542ee10e-39de-46b6-8ddc-e97d2e9efac9.png">